external_deps/build.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 
-# Exit on error
-# Error on undefined variable
-set -e
-set -u
+# Exit on undefined variable and error.
+set -u -e -o pipefail
```

Comment:
Have you fully tested this? I seem to recall one of our scripts not working with pipefail

Reply:
That's the first time I've heard that pipefail can be removed on purpose? What was the problem? If a script fails with pipefail that should be because of a real error…

Reply:
I don't remember but you can imagine scenarios that aren't a real error like a `grep` with 0 matches

Reply:
Weird, I probably never faced myself a scenario with a grep with zero match…

Reply:
Basically I usually do:

`if something | grep`

And that's not an error, since it is meant to be caught by `if`.

Reply:
If an exception is found, I prefer someone to put an explicit `|| true` on the exception than entirely removing the check for the whole script.

Reply:
It must not have been this one since there is only a single pipe in the entire script lol. LGTM